### ISSUE TITLE
ImFlow::ImFlowNode::update() causing window not to serialize to imgui.ini

### ISSUE THREAD

**Issue by PapaNaxos** [Fattorino/ImNodeFlow#24]
When calling `ImFlow::ImNodeFlow::update()` inside an ImGui window, that window no longer updates its position and size within `imgui.ini`

## Minimal reproduction
```cpp
static ImFlow::ImNodeFlow myGrid;
if (ImGui::Begin("Test Window"))
{
    myGrid.update();
}
ImGui::End();
```

Reposition/resize the "Test Window" and restart the application. The windows' new position/size will not be retained between executions.

If the call to `update()` is removed, the retention of the windows position/size behaves normally again.

| ImGui Info | |
|---|---|
| Version | `1.90 WIP` |
| Num | 18993 |

**Comment by PapaNaxos** [comment]
Actually it seems to be failing to save **ANYTHING** to `imgui.ini` when I use the `update()` method, even the positions of other windows.

**Comment by Fattorino** [comment]
My only guess is that the second context for the node rendering messes ImGui up. I'll have to do some testing on my end. Such a weird bug tho

**Comment by PapaNaxos** [comment]
Are you able to recreate the bug on your side?

I was also a bit suspicious of the second context, but didn't look into it too deeply. What's the reason for making a second context?

**Comment by Fattorino** [comment]
I'll try to recreate the issue in the following days, also the second context is needed for the zoom

**Comment by Fattorino** [comment]
Fixed with commit 3e4ed6e5b51cc9a874480b94f112d5965b0412b0
If needed, open another issue
